<?xml version="1.0" encoding="UTF-8"?>
<launch>
    <include file="$(find rtabmap_ros)/launch/stereo_mapping.launch">
        <arg name="rtabmapviz" value="false" /> 
        <arg name="rtabmap_args" value="-d --Grid/RangeMax 0 --Grid/CellSize 0.5 --Grid/ClusterRadius 2 --Grid/FromDepth true" />
        <arg name="rviz" value="false" />
        <arg name="stereo_namespace" value="/stereo_camera" />
        <arg name="approx_sync" value="true" />
        <arg name="visual_odometry" value="true"/>
        <arg name="odom_frame_id"           default="base_link"/>
        
        <arg name="subscribe_scan_cloud" value="true"/>         <!-- Assuming 3D scan if set -->
        <arg name="scan_cloud_topic" value="/lidar/points2"/>
        
        <!-- <arg name="visual_odometry"         default="false"/>          Generate visual odometry -->
        <!-- <arg name="odom_topic"              default="/odom"/>         Odometry topic used if visual_odometry is false -->
    </include>

    <!-- <node name="rviz" pkg="rviz" type="rviz" args="-d $(find safer_stack)/rviz/rtabmap_stereo.rviz" output="screen"/> -->
</launch>